kroxylicious-kms-provider-aws-kms-test-support/src/main/java/io/kroxylicious/kms/provider/aws/kms/model/CreateAliasRequest.java

```diff
@@ -10,10 +10,8 @@
 
 import com.fasterxml.jackson.annotation.JsonProperty;
 
-import edu.umd.cs.findbugs.annotations.NonNull;
-
-public record CreateAliasRequest(@JsonProperty("TargetKeyId") @NonNull String targetKeyId,
-                                 @JsonProperty("AliasName") @NonNull String aliasName) {
+public record CreateAliasRequest(@JsonProperty("TargetKeyId") String targetKeyId,
```

Comment:
These should really @JsonProperty(value = "TargetKeyId", required = "true"), but again this was a preexisting issue.